lib/kafka_ex/protocol/metadata.ex

```diff
@@ -1,5 +1,6 @@
 defmodule KafkaEx.Protocol.Metadata do
   alias KafkaEx.Protocol
+  import KafkaEx.Protocol.Common
```

Comment:
That argument makes sense.  As a matter of personal preference I still like to avoid `import` because it obfuscates the source of the imported functions.  FWIW my opinion is informed by trying to get other developers to adopt Elixir and observing what frustrates them.  

Either way, I'm just saying this for the sake of conversation, really.  I don't expect you to change it unless maybe @bjhaid has a strong opinion on the topic but I suspect he would have expressed it if he did ;)